examples/ios/objc/Puzzle/Controllers/RLMViewController.m

```diff
@@ -0,0 +1,249 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2015 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import "RLMViewController.h"
+#import <Realm/Realm.h>
+#import <Realm/RLMRealmConfiguration+Sync.h>
+#import <Realm/RLMSyncConfiguration.h>
+#import "RLMPuzzle.h"
+#import "RLMPuzzlePiece.h"
+#import "RLMPuzzleView.h"
+#import "RLMStartView.h"
+
+static NSString *kRLMPuzzleUUID = @"7387fa06-dffa-4891-97e1-9e35c3898658";
+static CGFloat kRLMPuzzleCanvasMaxSize = 768.0f;
+
+@interface RLMViewController () <RLMPuzzleViewDelegate, UIAlertViewDelegate>
+
+@property (nonatomic, strong) RLMStartView *startView;
+@property (nonatomic, strong) RLMPuzzleView *puzzleView;
+@property (nonatomic, strong) NSMutableArray *puzzlePieces;
+
+@property (nonatomic, strong) RLMNotificationToken *notificationToken;
+
+@property (nonatomic, strong) RLMResults *puzzles;
+@property (nonatomic, strong) NSString *currentPuzzleID;
```

Comment:
Looks like `currentPuzzleID` is only ever set to `kRLMPuzzleUUID`. I don't think this property is actually needed.